nixos/lib/testing-python.nix

```diff
@@ -256,16 +242,18 @@ rec {
           );
 
       driver = setupDriverForTest {
-        inherit testScript enableOCR skipLint passthru;
+        inherit enableOCR skipLint passthru;
+        testScript = mkTestScript pkgs.qemu_test;
         testName = name;
         qemu_pkg = pkgs.qemu_test;
-        nodes = nodes pkgs.qemu_test;
+        nodes = nodes pkgs.qemu_test true;
```

Comment:
It would be best if the `testScript` could reuse this `nodes` value instead; only invoking the configs an extra time, when `lib.any (n: n.config.virtualisation.useNixStoreImage) nodes`.